clients/cloud/ruby/README.md

```diff
@@ -0,0 +1,68 @@
+# Overview
+
+Produce messages to and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/) using the [ ZenDesk Ruby Client for Apache Kafka](https://github.com/zendesk/ruby-kafka).
+
+
+# Prerequisites
+
+* [Bundler](https://bundler.io/) installed on your machine. Install via `gem install bundler`
+* Access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/) cluster
+* Local file with configuration parameters to connect to your Confluent Cloud instance ([how do I find those?](https://docs.confluent.io/current/cloud/using/config-client.html#librdkafka-based-c-clients)). Format the file as follows:
+    ```bash
+    $ cat ~/.ccloud/ccloud.config
+    bootstrap.servers=<broker-1,broker-2,broker-3>
+    sasl.username=<api-key-id>
+    sasl.password=<secret-access-key>
+    ```
+* Install gems
+    ```bash
+    $ bundle install
```

Comment:
What directly should this be run from?

Reply:
From the `/ruby` directory. I'm unsure on how to best specify this

Reply:
Add it as a bash step before bundle install, assuming being in repo top dir:
`$ cd clients/ruby`  or whever it is

Reply:
This readme is located in `clients/ruby`. Wouldn't users read it from there?